src/tools/plan-generator.ts

```diff
@@ -139,6 +146,37 @@ export class PlanGenerator {
     return this.createHandleRegistry.get(handle);
   }
 
+  /** Generates Kotlin `Capabilities` from Capabilities. */
+  createCapabilities(capabilities: Capabilities): string {
+    const ktCapabilities  = [];
+
+    if (capabilities.isEmpty()) {
```

Comment:
The default is TiedToRuntime if no argument is passed. Won't this break existing code by forcing the second parameter to be empty?

Reply:
There's code above that checks for this. If the capabilities returned match Empty, then we don't add a second argument. 

in runtime/recipe/handle.ts, the default capability is empty.